Files/style.css

```
@@ -0,0 +1,244 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 100%;
+}
```

This is unnecessary, [this](https://www.paulirish.com/2012/box-sizing-border-box-ftw/) is what you intended.
```suggestion
*,
*::before,
*::after {
  box-sizing: inherit;
}

html {
  box-sizing: border-box;
}
```